example/hyprland.conf

```diff
@@ -1,88 +1,117 @@
+# Create your files separately and then link them to this file like this:
+# source = ~/.config/hypr/myColors.conf
+
+
+################
+### MONITORS ###
+################
 
 # See https://wiki.hyprland.org/Configuring/Monitors/
 monitor=,preferred,auto,auto
 
 
-# See https://wiki.hyprland.org/Configuring/Keywords/ for more
-
-# Execute your favorite apps at launch
-# exec-once = waybar & hyprpaper & firefox
+###################
+### MY PROGRAMS ###
+###################
 
-# Source a file (multi-file configs)
-# source = ~/.config/hypr/myColors.conf
+# See https://wiki.hyprland.org/Configuring/Keywords/
 
 # Set programs that you use
 $terminal = kitty
 $fileManager = dolphin
 $menu = wofi --show drun
 
-# Some default env vars.
-env = XCURSOR_SIZE,24
-env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
 
-# For all categories, see https://wiki.hyprland.org/Configuring/Variables/
-input {
-    kb_layout = us
-    kb_variant =
-    kb_model =
-    kb_options =
-    kb_rules =
+#################
+### AUTOSTART ###
+#################
 
-    follow_mouse = 1
+# Autostart necessary processes (like notifications daemons, status bars, etc.)
+# Or execute your favorite apps at launch like this:
 
-    touchpad {
-        natural_scroll = false
-    }
+# exec-once = $terminal
+# exec-once = nm-applet &
+# exec-once = waybar & hyprpaper & firefox
 
-    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
-}
 
-general {
-    # See https://wiki.hyprland.org/Configuring/Variables/ for more
+#############################
+### ENVIRONMENT VARIABLES ###
+#############################
+
+# See https://wiki.hyprland.org/Configuring/Environment-variables/
 
+env = XCURSOR_SIZE,24
+env = QT_QPA_PLATFORMTHEME,qt5ct # change to qt6ct if you have that
+env = XDG_CURRENT_DESKTOP,Hyprland
+env = XDG_SESSION_TYPE,wayland
+env = XDG_SESSION_DESKTOP,Hyprland
+
+
+#####################
+### LOOK AND FEEL ###
+#####################
+
+# Refer to https://wiki.hyprland.org/Configuring/Variables/
+
+# https://wiki.hyprland.org/Configuring/Variables/#general
+general { 
+    
     gaps_in = 5
     gaps_out = 20
+
     border_size = 2
+
+    # https://wiki.hyprland.org/Configuring/Variables/#variable-types for info about colors
     col.active_border = rgba(33ccffee) rgba(00ff99ee) 45deg
     col.inactive_border = rgba(595959aa)
 
-    layout = dwindle
+    # Set to true enable resizing windows by clicking and dragging on borders and gaps
+    resize_on_border = false 
 
     # Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
     allow_tearing = false
+
+    layout = dwindle
 }
 
+# https://wiki.hyprland.org/Configuring/Variables/#decoration
 decoration {
-    # See https://wiki.hyprland.org/Configuring/Variables/ for more
-
+    
     rounding = 10
 
+    # Change transparency of focused and unfocused windows
+    active_opacity = 1.0
+    inactive_opacity = 1.0
+
+    drop_shadow = true
```

Comment:
why was this moved? what's the point?

Reply:
no particular reason, just prettier (leaves `blur` section last, since it is a subsection)